lib/cfe_internal_hub.cf

```diff
@@ -209,17 +209,19 @@ bundle agent cfe_internal_database_cleanup_consumer_status (row_count)
         expression => returnszero( "$(sys.bindir)/psql cfdb -c \"SELECT 'cleanup_historical_data'::regproc;\" > /dev/null 2>&1", useshell ),
         if => isexecutable( "$(sys.bindir)/psql" );
 
+      "has_sql_table___status" -> { "ENT-4331" }
+        expression => returnszero( "$(sys.bindir)/psql cfdb -c \"SELECT 'public.__schema'::regclass;\" > /dev/null 2>&1", useshell ),
```

Comment:
I would change this variable to cfdb_schema since that's what it is and you grep it later on.

Reply:
Yeah, this is really confusing.

Reply:
fixed, changed to __status

Reply:
Also fixed in other PRs.